mobile/lib/presentation/widgets/bottom_sheet/general_bottom_sheet.widget.dart

```diff
@@ -25,14 +26,15 @@ import 'package:immich_mobile/providers/server_info.provider.dart';
 import 'package:immich_mobile/providers/timeline/multiselect.provider.dart';
 import 'package:immich_mobile/widgets/common/immich_toast.dart';
 
-class GeneralBottomSheet extends ConsumerWidget {
+class GeneralBottomSheet extends HookConsumerWidget {
```

Comment:
Can you convert this to `StatefulComsumerWidget` to avoid using hooks?

Reply:
done!